### ISSUE TITLE
Newly cataloged for selectors: Allow greater complexity in combining call numbers with keywords

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/lib_jobs#591]
### User story
Jeremy Darrington would like to more tightly scope the results he gets. He would like:
- All books in the J* classes (with the existing exceptions of ones published in the US, etc.)
- Books with the keyword "political" in the subjects, but *only* if they are also in the following classes: [D, F, G, H, K, or U]

### Implementation notes, if any
- This could end up being a very complex feature to implement.

**Comment by mzelesky** [comment]
I agree that this could end up being very complex. I would say that if a selector just included all the classes, they could filter out the ones they aren't interested in purchasing by looking at subject headings.
